imports/plugins/core/shipping/client/containers/parcelSizeSettingContainer.js

```diff
@@ -0,0 +1,99 @@
+import { compose, withProps } from "recompose";
+import { registerComponent, composeWithTracker } from "@reactioncommerce/reaction-components";
+import { Meteor } from "meteor/meteor";
+import { Reaction } from "/client/api/";
+import { isEmpty } from "lodash";
+import { Shops } from "/lib/collections";
+import ParcelSizeSettings from "../components/parcelSizeSettings";
+
+/**
+ * @method isNumber
+ * @summary checks if an input is an integer or a decimal
+ * @param {String} input - input string
+ * @since 1.5.5
+ * @return {Boolean} - returns a boolean result
+ */
+const isNumber = (input) => {
```

Comment:
The description and the name don't match. Does this tell you if something is a number or if it's an integer or a decimal (both numbers)? If it's checking if it's a number or not I would prefer we use `_.isNumber` or `_.isFinite` rather than writing our own one-off batch of regex.

Reply:
@zenweasel the function was written to check that the input is not a string. Initially, I tried `_.isNumber` but it also returned true for a string. I'll try the `_.isFinite` method.